crypto/cms/cms_ec.c

```diff
@@ -60,6 +63,7 @@ static EVP_PKEY *pkey_type2param(int ptype, const void *pval,
  err:
     EVP_PKEY_free(pkey);
     EVP_PKEY_CTX_free(pctx);
```

Comment:
Why doesnt the pctx get freed above in it does return pkey?

Reply:
No idea.  @DDvO ?  @mattcaswell ?

Reply:
Simply a bug (a leak). I suppose we aren't testing that case.

Reply:
I see this code for the first time, but the missing `EVP_PKEY_CTX_free(pctx);`
at the end of the first `else` block is clearly an oversight.

Reply:
I've added the free call.